Shaders/CinematicDOF.fx

```diff
@@ -442,17 +445,17 @@ namespace CinematicDOF
 	// returns minCoC
 	float PerformTileGatherHorizontal(sampler source, float2 texcoord)
 	{
-		float tileSize = TILE_SIZE * (float(BUFFER_WIDTH) / float(GROUND_TRUTH_SCREEN_WIDTH));
+		float tileSize = TILE_SIZE * (ReShade::ScreenSize.x / GROUND_TRUTH_SCREEN_WIDTH);
 		float minCoC = 10;
 		float coc;
-		float2 coordOffset = float2(BUFFER_PIXEL_SIZE.x, 0);
+		float2 coordOffset = float2(ReShade::PixelSize.x, 0);
```

Comment:
It's better to use the new `BUFFER_PIXEL_SIZE` and `BUFFER_SCREEN_SIZE` defines here. This ensures better compatibility with FreeStyle (because the screen/pixel size is not a constant in FreeStyle, so the ReShade.fxh has some ugly workarounds to have `ReShade::ScreenSize`/`ReShade::PixelSize` look like global constant variables anyway, which I'd much rather get rid of in the future).

Reply:
Btw., you can also check whether an effect is compiled for FreeStyle by looking whether `__RESHADE_FXC__` is defined. That way you don't need to maintain two different effect variants and can instead have it all in one file.